redirects.json

```diff
@@ -179,7 +179,11 @@
     { "destination": "/docs/intro/wallets", "source": "/wallet-guide/support" },
     { "destination": "/docs/intro/wallets", "source": "/wallet-guide" },
     { "destination": "/docs/intro/wallets", "source": "/wallet-guide/:path" },
-    { "destination": "/docs/intro", "source": "/introduction" }
+    { "destination": "/docs/intro", "source": "/introduction" },
+    {
+      "destination": "/docs/rpc/http/getstakeactivation",
```

Comment:
the setup a redirect, it should there should be an `altRoutes` entry added into the frontmatter of the getStakeActivation file, not this redirects.json file (confusing/annoying I know)

like this: https://github.com/solana-foundation/developer-content/blob/main/docs/rpc/deprecated/getConfirmedTransaction.mdx?plain=1#L5-L7

Reply:
update: the correct `altRoutes` entry is already made for this deprecated RPC method. you don't need to add anything here